package.json

```diff
@@ -7,16 +7,19 @@
   ],
   "devDependencies": {
     "@changesets/cli": "^2.27.7",
+    "@eslint/js": "^9.8.0",
     "depcheck": "^1.4.7",
-    "eslint": "^8.57.0",
+    "eslint": "^9.8.0",
```

Comment:
any particular reason to use different versions from https://github.com/nordeck/matrix-widget-toolkit/pull/738?

Reply:
Yes, because a newer version is available.
Also a ticket exist, so move the config and dependencies to the toolkit (or an eslint repo)